ui/src/pages/backoffice/Series/SeriesSearch/SeriesListEntry.js

```diff
@@ -34,82 +35,43 @@ export class SeriesListEntry extends Component {
 
   renderRelations = () => {
     const { series } = this.props;
-    const serialsCount = _get(series, 'metadata.relations_metadata.serial', [])
-      .length;
-    const monographsCount = _get(
+    // create queryString to find all MM or Series related to this series
+    const partOfMMQuery = _get(
       series,
-      'metadata.relations_metadata.multipart_monograph',
+      'metadata.relations.multipart_monograph',
       []
-    ).length;
-    const documentsCount = serialsCount + monographsCount;
-    const seriesType =
-      series.metadata.relations_metadata &&
-      series.metadata.relations_metadata.multipart_monograph
-        ? 'multipart_monograph'
-        : 'serial';
+    )
+      .map(rel => `pid:${rel.pid_value}`)
+      .join(' OR ');
+    const partOfSeriesQuery = _get(series, 'metadata.relations.serial', [])
+      .map(rel => `pid:${rel.pid_value}`)
+      .join(' OR ');
     return (
       <>
         <Item.Description>
           <List verticalAlign={'middle'} className={'series-relations'}>
-            {!_isEmpty(series.metadata.relations.serial) && (
+            {partOfMMQuery && (
               <List.Item>
-                <List.Content>
+                <List.Content floated="right">
```

Comment:
@kprzerwa any opinion on this?

Reply:
you probably mean that there is styling here?
I'm ok with that, because it is a semantic-ui prop, so IMO it is correct to first change it by props and then only as a last resort to overwrite the theme (which is not needed here)